filebeat/module/mysql/error/test/mysql-ubuntu-5.5.53-extented.log-expected.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "@timestamp": "2023-03-12T19:26:00.000-02:00",
+        "event.category": [
+            "database"
```

Comment:
Is this event also 'network'?

Reply:
I suppose it is and maybe connection too

Reply:
In `event.type`, yeah, probably.

Reply:
fixed pipeline and added network event.category. event.type is info cause connection require source.ip and destination.ip